content/usgs-packages/dataRetrieval_readNWIS.Rmd

```diff
@@ -284,17 +300,16 @@ makeArgDefTable("readNWISpeak", table.num=7)
 
 **Peak flow values for a site near Cassia, Florida.**
 
-The default settings will return data where the date of the peak flow is known. To include peak flows with unknown dates, change `asDateTime` to `FALSE`. This will keep all rows of the data.
+The default settings will return data where the date of the peak flow is known. To see peak flows with incomplete dates, change `convertType` to `FALSE`. 
 
 ```{r readnwispeak, message=FALSE, warning=FALSE}
 # Major filter: site number, 02235200
 fl_site_peak <- readNWISpeak(siteNumbers="02235200")
-head(fl_site_peak)
+fl_site_peak$peak_dt
 
 # Compare complete with incomplete days
-nrow(fl_site_peak)
-fl_site_peak_incomp <- readNWISpeak(siteNumbers="02235200", asDateTime=FALSE)
-nrow(fl_site_peak_incomp)
+fl_site_peak_incomp <- readNWISpeak(siteNumbers="02235200",convertType = FALSE)
+fl_site_peak_incomp$peak_dt[is.na(fl_site_peak$peak_dt)]
```

Comment:
I think this line should have a comment explaining what it's doing. 

I like what you've added - much more explicit about what we differences we are trying to get people to notice.